caffe2/CMakeLists.txt

```diff
@@ -1320,8 +1320,11 @@ if(BUILD_TEST)
         list(GET CPU_CAPABILITY_NAMES ${i} CPU_CAPABILITY)
         list(GET CPU_CAPABILITY_FLAGS ${i} FLAGS)
         separate_arguments(FLAGS UNIX_COMMAND "${FLAGS}")
-        add_executable(${test_name}_${CPU_CAPABILITY} "${test_src}")
-        target_link_libraries(${test_name}_${CPU_CAPABILITY} torch_library gtest_main)
+        add_executable(${test_name}_${CPU_CAPABILITY} "${test_src}" ../aten/src/ATen/native/quantized/affine_quantizer_base.cpp)
```

Comment:
noob question: shouldn't affine_quantizer_base.cpp already been included in `ATen_QUANTIZED_SRCS`? why do we need to manually included it again here?

Reply:
@walterddr because  it explicitly tests that vec256 does not depend on some random function of libtorch or libc10 or CUDA. But be explicit of that constraint, I've removed `torch_library` dependency.